### ISSUE TITLE
Indent rule with multi-line object literals

### ISSUE THREAD

**Issue by silverwind** [eslint/eslint#4075]
Below example should pass, but the rule seems to get confused by the object literal. It passes in 0.24, fails in 1.6.0. [Real example](https://github.com/nodejs/node/blob/a334ddc467af6bfb3a93245a7e6f571c8dd01fa2/test/sequential/test-regress-GH-4948.js#L19-L27).
````
indent: [2,2]
````
````js
require('http').request({hostname: 'localhost',
                         port: 80}, function(res) {
  res.end();
});
````
````
  3:3  error  Expected indentation of 27 space characters but found 2  indent
  4:1  error  Expected indentation of 25 space characters but found 0  indent
````
ref: https://github.com/nodejs/node/pull/2286

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Its the function body getting confused because of the multi-line argument.
related https://github.com/eslint/eslint/issues/4073
